_posts/2017-02-20-five-tips-to-empower-content-authors-using-drupal-8.md

```diff
@@ -0,0 +1,147 @@
+
+Here’s 5 of them.
+
+[1. Don’t make authors guess - use structured content](#dont-make-authors-guess---use-structured-content)
+
+[2. Configure WYSIWYG editor responsibly](#configure-wysiwyg-editor-responsibly)
+
+[3. Empower your editorial team with Quick-Edit](#empower-your-editorial-team-with-quick-edit)
+
+[4. Enriched content with Media Embeds](#enriched-content-with-media-embeds)
+
+[5. Simplify content linking with LinkIt](#simplify-content-linking-with-linkit)
+
+## 1. Don’t make authors guess - use structured content
+
+The abundance of different devices, screen sizes and form factors warrants the use of structured content. Structured content is content separated into distinct parts, each of which has a clearly defined purpose and can be edited and presented independently from one another according to context.
+
+“How does that relate to a content editor’s experience?” - you may ask.
+
+In years past, it was very popular to give content editors an ability to create “pages” using one big “MS Word-like” text box for writing their articles, news releases, product descriptions, etc. This approach produced content that was not re-usable and was presented in one strict way.
+
+Well, those days are long behind us and even though we all know about the importance of structured content, sometimes we still fail to utilize the concept correctly. Plus, who wants to navigate within the enormous text area to move images or text around?
+
+The concept of structured content is not new to Drupal. In fact Drupal was one of the first Content Management Systems (CMS) to introduce it (node system - Drupal 3 in 2001). Drupal is no-doubt the best CMS for implementing the concept of structured content, but it always lacked in its ability to provide a good content authoring experience.
+
+In Drupal 8, editing structured content is a joy! With Wysiwyg (What you see is what you get) editor and Quick Edit functionality already in Drupal core, we can equip our content editors with the best of class authoring experience and workflow!
+
+To visualize the difference between unstructured and structured content - see the image below.
+
+<img src="/assets/img/blog/structured-vs-unstructured.jpg" class="blog-image-large" alt="Structured vs unstructured content">
+
+Unstructured content has only one field which contains text, images, etc. Alternatively, structured content allows to store each definitive piece of information in it’s own field, making content entry fast and presentation flexible.
+
+### The benefits of Drupal 8 structured content approach:
+
+- The content author clearly understands where each piece of information should reside. Content entry becomes remarkably efficient.
+- The content author does not have to factor in the markup, layout and design of the content while editing it  (see [tip #2](/#configure-wysiwyg-editor-responsibly) below). This enables the author to concentrate on the essence of their message instead of content format and presentation.
```

Comment:
@oksana-c This is the other one that needs to be fixed, just need to remove that `/`!